test/transaction.ts

```diff
@@ -61,6 +62,9 @@ describe('Transaction', () => {
     request_() {},
     projectId: PROJECT_ID,
     namespace: NAMESPACE,
+    addDatabaseIdToRequest(d: any, r: any) {
```

Comment:
I would expect this to be: `databaseId: DATABASE` similar to project and namespace. Can you help me understand this better?